.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Should likely add `.vscode` to `.gitignore`

Reply:
If we commit it, then everyone who is using .vscode with this repo will have their editor configured to use the local TypeScript installation for syntax highlighting, vs whichever one they have built in to their version of VSCode

Reply:
Okay, that's a benefit. To discuss with the team